packages/otest/lib/Assertion.ts

```diff
@@ -0,0 +1,186 @@
+import { TestError, TestResult } from "./TestResult.js"
+
+export type AssertionDescriber = (description: string) => void
+const noop = () => {}
+
+class AssertionError extends Error {}
+
+interface Class<T> {
+	new (...args: any[]): T
+}
+
+function xor(a: boolean, b: boolean): boolean {
+	return (a && !b) || (b && !a)
+}
+
+function isArguments(a: any) {
+	if ("callee" in a) {
+		for (let i in a) if (i === "callee") return false
```

Comment:
i looks like an iteration variable (a number) but seems it can be anything? also `const i`